File: packages/aws-cdk/lib/init-templates/app/dotnet/src/HelloCdk/HelloStack.cs
using Amazon.CDK;
using Amazon.CDK.AWS.SNS;
using Amazon.CDK.AWS.SQS;
using System;
using System.Collections.Generic;
using System.Text;

namespace HelloCdk
{
    public class HelloStack : Stack
    {
        public HelloStack(App parent, string name, IStackProps props) : base(parent, name, props)
        {
            var queue = new Queue(this, "MyFirstQueue", new QueueProps
            {
                VisibilityTimeoutSec = 300
            });

            var topic = new Topic(this, "MyFirstTopic", new TopicProps
            {
                DisplayName = "My First Topic Yeah"
            });

            topic.SubscribeQueue(queue);
        }
    }
}

File: packages/aws-cdk/lib/init-templates/app/fsharp/src/HelloCdk/HelloStack.fs
namespace HelloCdk

open HelloCdk

open Amazon.CDK
open Amazon.CDK.AWS.IAM
open Amazon.CDK.AWS.SNS
open Amazon.CDK.AWS.SNS.Subscriptions
open Amazon.CDK.AWS.SQS

type MyStack(scope, id, props) as this =
    inherit Stack(scope, id, props)

    let queue = Queue(this, "MyFirstQueue", QueueProps(VisibilityTimeout = Duration.Seconds(300.)))

    let topic = Topic(this, "MyFirstTopic", TopicProps(DisplayName = "My First Topic Yeah"))
    do topic.AddSubscription(SqsSubscription(queue, null))

    let hello = HelloConstruct(this, "Buckets", { BucketCount = 5 })
    let user = User(this, "MyUser", UserProps())
    do hello.GrantRead(user)
